static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain.js

```diff
@@ -88,9 +93,12 @@ export const getOutbrainComplianceTargeting = (): Promise<
   true              false               false             n/a              false         false      true      non-compliant
   true              false               false             n/a              false         false      false     compliant
 */
+
+export const shouldTestOutbrainWidget = isInOutbrainTestingVariant();
```

Comment:
we probably do not need to export this since it is not being used in any other modules 😄